custom-widget-builder/src/Property.ts

```diff
@@ -24,12 +24,12 @@ export interface Property {
   name: string;
   type: PropertyType | undefined;
   label: string | undefined;
-  help: string | undefined;
+  help?: string | undefined;
```

Comment:
If you add the interrogation mark (optional), then 'undefined' is not needed anymore.
And also align all the properties with undefined as an optional.